test/benchmarks/memory_usage.js

```diff
@@ -105,7 +108,7 @@ class MemoryBenchamrkSuite {
  * Memory usage tests.
  * All the necessary variables must be declared inside the test function.
  */
-(async () => {
+void (async () => {
```

Comment:
If this is to make the linter happy, could we just add a comment instead, to make that explicit? Adding code to make the linter happy feels very silly IMHO..
(below as well)